File: src/main/java/com/back/mozu/domain/admin/dto/AdminLoginResponseDto.java
package com.back.mozu.domain.admin.dto;


import lombok.Builder;
import lombok.Getter;

import java.util.UUID;

@Getter
@Builder
public class AdminLoginResponseDto {
    private String accessToken;
    private String refrestToken;
    private AdminUserDto adminUser;

    @Getter
    @Builder
    public static class AdminUserDto{
        private String adminId;
        private String loginId;
        private String name;
    }
}

File: src/main/java/com/back/mozu/domain/admin/dto/AdminLoginResponseDto.kt
package com.back.mozu.domain.admin.dto

data class AdminLoginResponseDto(
    val accessToken: String,
    val refreshToken: String,
    val adminUser: AdminUserDto,
) {
    data class AdminUserDto(
        val adminId: String,
        val loginId: String,
        val name: String,
    )
}